keentools/ui_strings.py

```diff
@@ -186,4 +186,14 @@
         'If you want to manually update the add-on: remove the old add-on, ',
         'restart Blender and install the new version of the add-on.'
     ]),
+    ErrorType.FBGracePeriod: ErrorMessage(_default_width, [
+        'Your FaceBuilder License is switched to Grace period',
```

Comment:
@artyomxx please check.

Reply:
> Sorry, there's a problem with your subscription.
> 
> If you're using 'Online' mode, please check that your payment method is still valid and you have sufficient funds on your account. We'll try to charge the subscription fee again in the next couple of days.
> 
> If you're using 'Offline' mode and have generated the license file manually, it's time to renew the file. Please log in to User Portal, download a new file and activate it as you did before.

@bbugaev pls, no talks of "there's no online/offline mode" anymore:

<img width="243" alt="image" src="https://user-images.githubusercontent.com/888428/207298863-bd770736-d024-4c90-b640-9965b2a886d2.png">

<img width="330" alt="image" src="https://user-images.githubusercontent.com/888428/207298965-f1d7c94b-8553-4870-8fdd-fb4df961a935.png">